resources/user-agents/browsers/opera-mobile/opera-mobile.json

```diff
@@ -1,66 +0,0 @@
-{
```

Comment:
I brought all of the patterns in this file into the `opera-mobile-generic.json` file. It seemed unnecessary to have them separate.